### PR TITLE
Fix examples in "docker port" man page.

### LINKED ISSUE #13667: 'docker port' man page example uses wrong syntax
In man page for docker-port, the examples section says to type 'docker ps test' but currently, docker ps takes 0 arguments.

The examples should instead say 

    docker ps -a | grep test

### PR DESCRIPTION
**Pull request by draghuram**
Closes #13667.

Signed-off-by: Raghuram Devarakonda draghuram@gmail.com

### CODE REVIEW

**Comment by SvenDowideit** [comment]
LGTM

File: docs/man/docker-port.1.md [hunk 1]
```diff
@@ -17,22 +17,30 @@ List port mappings for the CONTAINER, or lookup the public-facing port that is N
...
-You can find out all the ports mapped by not specifying a `PRIVATE_PORT`, or
-ask for just a specific mapping:
 
-    $ docker ps test
+    # docker ps
     CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS                                            NAMES
     b650456536c7        busybox:latest      top                 54 minutes ago      Up 54 minutes       0.0.0.0:1234->9876/tcp, 0.0.0.0:4321->7890/tcp   test
-    $ docker port test
+
+# Find out all the ports mapped
```
Comment (thaJeztah) [inline docs/man/docker-port.1.md]: Can you change these to a H2 (`## Find out all the ports mapped`)?

File: docs/man/docker-port.1.md [hunk 2]
```diff
@@ -17,22 +17,30 @@ List port mappings for the CONTAINER, or lookup the public-facing port that is N
...
+
+    # docker port test
     7890/tcp -> 0.0.0.0:4321
     9876/tcp -> 0.0.0.0:1234
-    $ docker port test 7890/tcp
+
+# Find out a specific mapping
```
Comment (thaJeztah) [inline docs/man/docker-port.1.md]: Same here

File: docs/man/docker-port.1.md [hunk 3]
```diff
@@ -17,22 +17,30 @@ List port mappings for the CONTAINER, or lookup the public-facing port that is N
...
+    # docker port test 7890/tcp
     0.0.0.0:4321
-    $ docker port test 7890/udp
-    2014/06/24 11:53:36 Error: No public port '7890/udp' published for test
-    $ docker port test 7890
+
+    # docker port test 7890
     0.0.0.0:4321
 
+# An example showing error for non-existent mapping
```
Comment (thaJeztah) [inline docs/man/docker-port.1.md]: And here

**Comment by thaJeztah** [comment]
Thanks @draghuram, added a minor comment, but looks good to met after that's changed

**Comment by draghuram** [comment]
@thaJeztah I made changes to the PR. Please let me know if this is what you had in mind.

**Comment by thaJeztah** [comment]
Yup, that's what I meant!

LGTM, thanks!

**Pull request merged**